interni/verbali/verbale-interno_2020-04-13.tex

```diff
@@ -0,0 +1,100 @@
+\documentclass{article}
+
+\input{../../commons/config}
+
+\appendToGraphicspath{../../commons/img/}
+
+\title{Verbale interno --- 13/04/2020}
+
+\setResponsabile{Luca Ercole}
+\setRedattori{Alberto Cocco}
+\setVerificatori{
+  Alberto Gobbo
+}
+\setUso{Interno}
+\setDescrizione{Verbale dell'incontro di GruppOne del 13/04/2020}
+\setModifiche{%
+\cellcolor{white!80!lightgray!100} & Luca Ercole & 2020--04--14 & approva documento \\%
+\cellcolor{white!80!lightgray!100} & Alberto Gobbo & 2020--03--13 & verifica verbale \\%
+\multirow{-3}{*}{-} & Alberto Cocco & 2020--04--13 & stendi verbale%
+}
+
+\disabilitaVersione{}
+\disabilitaElencoFigure{}
+\disabilitaElencoTabelle{}
+
+\begin{document}
+
+\input{../../commons/template}
+
+\section{Informazioni logistiche}%
+\label{sec:informazioni_logistiche}
+
+\begin{description}
+  \item [Luogo] videochiamata su Google Hangouts
+  \item [Data] 13/04/2020
+  \item [Ora] 18:30 \symbol{8594} 20:00
+\end{description}
+
+\subsection{Membri del gruppo presenti}%
+\label{sub:membri_del_gruppo_presenti}
+
+\begin{enumerate}
+  \item Riccardo Agatea
+  \item Tobia Apolloni
+  \item Riccardo Cestaro
+  \item Alberto Cocco
+  \item Luca Ercole
+  \item Alberto Gobbo
+  \item Alessandro Rizzo
+  \item Fabio Scettro
+\end{enumerate}
+% sub:membri_del_gruppo_presenti (end)
+
+\section{Introduzione}%
+\label{sec:introduzione}
+
+L'incontro si è svolto su Google Hangouts per riorganizzare il lavoro in vista della revisione di progettazione.
+
+\section{Ordine del giorno}%
+\label{sec:ordine_del_giorno}
+\begin{itemize}
+  \item Istituzione di un documento di tracciamento dei requisiti.
+  \item Creazione di una kanban board.
+\end{itemize}
+
+\section{Documento di tracciamento dei requisiti}%
+\label{sec:documento_di_tracciamento_dei_requisiti}
+
+Per avere un controllo maggiore sullo stato di avanzamento del progetto si è deciso di creare un documento di tracciamento dei requisiti.
+Ogni requisito è stato riportato all'interno di un Google docs con la rispettiva sigla. Per ogni requisito si sono inoltre indicati i componenti(app, web app o server)
+coinvolti e l'incremento in cui il requisito deve essere soddisfatto.
```

Comment:
(...) di un Google _Docs_ con la rispettiva sigla. Per ogni requisito vengono indicati i componenti coinvolti(mobile application,  web application, server) e l'incremento in cui il requisito deve essere soddisfatto.